src/AzureSignTool/SignCommand.cs

```diff
@@ -345,6 +350,9 @@ private static ErrorOr<X509Certificate2Collection> GetAdditionalCertificates(IEn
             return collection;
         }
 
-        private static bool OneTrue(bool left, bool right) => left != right && (left || right);
+        private static bool OneTrue(params bool[] checks)
+        {
+	        return checks.Count(boolean => boolean == true) == 1;
```

Comment:
This could probably be simplified to:
```suggestion
	        return checks.Count(b => b) == 1;
```

Reply:
I had actually written the line exactly like you had it to begin with.  I was concerned with readability/clarity for future coders, so expanded it out to be clear that it's counting the true booleans.  But yeah, I can re-condense the change.